src/accessories/CampaignImporterExporter.ttslua

```diff
@@ -0,0 +1,302 @@
+    Name = "Custom_Model",
+    Transform = {
+      posX = 2.80123329,
+      posY = 1.681688,
+      posZ = -23.6498032,
+      rotX = -2.22745348E-06,
+      rotY = 269.999664,
+      rotZ = -1.6614282E-06,
+      scaleX = 2.00499868,
+      scaleY = 2.00499868,
+      scaleZ = 2.00499868
+    },
+    Nickname = "Arkham Coin",
+    Description = "SCED Importer Token",
+    GMNotes = "",
+    AltLookAngle = {
+      x = 0.0,
+      y = 0.0,
+      z = 0.0
+    },
+    ColorDiffuse = {
+      r = 1.0,
+      g = 1.0,
+      b = 1.0
+    },
+    Tags = {
+        "ImporterToken"
+    },
+    LayoutGroupSortIndex = 0,
+    Value = 0,
+    Locked = false,
+    Grid = true,
+    Snap = true,
+    IgnoreFoW = false,
+    MeasureMovement = false,
+    DragSelectable = true,
+    Autoraise = true,
+    Sticky = true,
+    Tooltip = true,
+    GridProjection = false,
+    HideWhenFaceDown = false,
+    Hands = false,
+    CustomMesh = {
+      MeshURL = "http://cloud-3.steamusercontent.com/ugc/943949966265929204/A38BB5D72419E6298385556D931877C0A1A55C17/",
+      DiffuseURL = "http://cloud-3.steamusercontent.com/ugc/254843371583188147/920981125E37B5CEB6C400E3FD353A2C428DA969/",
+      NormalURL = "",
+      ColliderURL = "http://cloud-3.steamusercontent.com/ugc/943949966265929204/A38BB5D72419E6298385556D931877C0A1A55C17/",
+      Convex = true,
+      MaterialIndex = 2,
+      TypeIndex = 0,
+      CustomShader = {
+        SpecularColor = {
+          r = 0.7222887,
+          g = 0.507659256,
+          b = 0.339915335
+        },
+        SpecularIntensity = 0.4,
+        SpecularSharpness = 7.0,
+        FresnelStrength = 0.0
+      },
+      CastShadows = true
+    },
+    LuaScript = "",
+    LuaScriptState = "",
+    XmlUI = ""
+}
+
+-- colors and order for following tables
+local COLORS = { "White"; "Orange"; "Green"; "Red"; "Agenda" }
+
+-- counter GUIDS (4x damage and 4x horror)
+local DAMAGE_HORROR_GUIDS = {
+    "eb08d6"; "e64eec"; "1f5a0a"; "591a45";
+    "468e88"; "0257d9"; "7b5729"; "beb964";
+  }
+
+local campaignData = {}
+
+local tokenZone = getObjectFromGUID("ef3b5f")
+local deckImporter = getObjectFromGUID("a28140")
+local investigatorCounter = getObjectFromGUID("f182ee")
+
+local campaignBoxGUID = ""
+local traumaValues = false
+local deckIds = false
+local campaignGuidePage = false
+
+function onLoad(save_state)
+    self.createButton({
+        click_function = "findCampaignFromToken",
+        function_owner = self,
+        label = "Import",
+        position = {x=-1, y=0.2, z=0},
+        width = 350,
+        height = 150,
+        scale = {2, 1, 2},
+    })
+    self.createButton({
+        click_function = "createCampaignToken",
+        function_owner = self,
+        label = "Export",
+        position = {x=1, y=0.2, z=0},
+        width = 350,
+        height = 150,
+        scale = {2, 1, 2},
+    })
+end
+
+function findCampaignFromToken(_, _, _)
+    local coin = nil
+    for _, obj in ipairs(tokenZone.getObjects()) do
```

Comment:
Any reason not to just use getObjectsWithTag here, and avoid both the zone and loop?